inst/ms/aggreCAT.qmd

```diff
@@ -223,10 +256,11 @@ types:
 -   Bayesian methods that use participant judgements as data with which
     to update both uninformative and informative priors.
 
-However, the [aggreCAT]{.pkg} package user might wish to categorise the aggregation methods according to aspects of
-their computational implementation and data requirements, because these
-inform the arguments and the type and form of the data that is parsed to
-the aggregation functions. These aspects include:
+However, the [aggreCAT]{.pkg} package user might wish to categorise the
+aggregation methods according to aspects of their computational
+implementation and data requirements, because these inform the function
+arguments as well as the type and form of the data that is parsed to the
+aggregation functions. These aspects include:
 
 -   Elicitation Method, number of elicitation rounds: the majority of
```

Comment:
I think "Elicitation Method, number of elicitation rounds" is a bit clunky and not all that clear. Maybe "Elicitation requirement (number of rounds)"? Next point then becomes "Elicitation requirement (estimate type)" or similar?